rocAL/rocAL/source/meta_data/tf_meta_data_reader_detection.cpp

```diff
@@ -77,14 +77,14 @@ void TFMetaDataReaderDetection::lookup(const std::vector<std::string> &image_nam
 	
         if(_map_content.end() == it)
         {
-            _output->get_bb_cords_batch()[i] = {{0, 0, 0, 0}};
-            _output->get_bb_labels_batch()[i] = {{0}};
+            getMetaDataBatchValues<std::vector<BoundingBoxCords>>(*_output, &MetaDataBatch::get_bb_cords_batch)[i] = {{0, 0, 0, 0}};
+            getMetaDataBatchValues<std::vector<Labels>>(*_output, &MetaDataBatch::get_labels_batch)[i] = {{0}};
             _output->get_img_sizes_batch()[i] = {0, 0};
         }
         else
         {
-            _output->get_bb_cords_batch()[i] = it->second->get_bb_cords();
-            _output->get_bb_labels_batch()[i] = it->second->get_bb_labels();
+            getMetaDataBatchValues<std::vector<BoundingBoxCords>>(*_output, &MetaDataBatch::get_bb_cords_batch)[i] = getMetaDataValues<BoundingBoxCords>(*it->second, &MetaData::get_bb_cords);
+            getMetaDataBatchValues<std::vector<Labels>>(*_output, &MetaDataBatch::get_labels_batch)[i] = getMetaDataValues<std::vector<int>>(*it->second, &MetaData::get_labels);
```

Comment:
Do we not have a struct that can be used instead of std::vector<int> for labels similar to BoundingBoxCords ?